### ISSUE TITLE
wpcf7_add_shortcode deprecated since Contact Form 7 version 4.6

### ISSUE THREAD

**Issue by arnelap** [ibericode/mailchimp-for-wordpress#554]
wp-content/plugins/mailchimp-for-wp/integrations/contact-form-7/class-contact-form-7.php:42 calls wpcf7_add_shortcode, which is deprecated since Contact Form 7 version 4.6! Use wpcf7_add_form_tag instead.

https://wordpress.org/support/topic/plugin-uses-deprecated-cf7-code/

**Comment by dannyvankooten** [comment]
![image](https://user-images.githubusercontent.com/885856/39467953-f549a928-4d30-11e8-9a6f-5c515da82db3.png)

This function is only called if the newer function is not available, so should not be an issue.

**Comment by maxxwv** [comment]
I'm using the most recent version of both this plugin and Contact Form 7 and it's still an issue. Disabling this plugin removes the error, but obviously I'd rather not have to do that. Can this issue be re-opened?

**Comment by arnelap** [comment]
Hi maxxwv,

If the issue is closed here it means the code is written and added to the master branch. It can take a bit longer before we publish it as an official update. 

If you can't wait for that, you can install the development version of the plugin by downloading the zip file from here: https://github.com/ibericode/mailchimp-for-wordpress

You can scroll down on that page for the readme to get more information on how to do that. 

Hope that helps. If you have any questions, please let us know!

**Comment by dannyvankooten** [comment]
This issue was fixed in late 2016, via c69ce2c5f2e6d6a160c872b08d87ba9f5bbff30e . So nothing changed in the last few weeks.

Is the issue still occurring for you @maxxwv? Or is it gone and where you on a relatively old version of the plugin?

Thanks in advance!

**Comment by maxxwv** [comment]
Sorry for the confusion, all. The plugin reported being on the latest version on the Plugins screen, but when I pulled a copy locally and looked at the code it was obvious that it wasn't actually the latest version - the code quoted above didn't have the conditional check. It looks like the update failed due to the error reporting sending output headers in AJAX. I rolled back to an older version, turned off error reporting, and updated successfully so everything is working now. Again, sorry about the confusion and thanks for the quick replies!
